packages/perseus-score/src/widgets/dropdown/score-dropdown.ts

```diff
@@ -10,10 +8,6 @@ function scoreDropdown(
     userInput: PerseusDropdownUserInput,
     rubric: PerseusDropdownRubric,
 ): PerseusScore {
-    const validationError = validateDropdown(userInput);
```

Comment:
Jeremy made a change that calls all validators before calling any scorers, so we don't need to start these functions with validation anymore.